base_location/models/company.py

```diff
@@ -6,19 +6,22 @@
 
 
 class ResCompany(models.Model):
-
     _inherit = 'res.company'
 
+    better_zip_id = fields.Many2one(
```

Comment:
This shouldn't be needed as company_id is inherited by delegation of res.partner. And please, don't call it `better_zip_id`.